packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -422,6 +422,7 @@ const ReleaseDetailsLayout = ({
                     defaultMessage: 'Release edit and delete menu',
                   })}
                   variant="tertiary"
+                  endIcon={null}
```

Comment:
Should the comment above the updated?

Reply:
Thanks! take the opportunity to change this to the SimpleMenu component that has what we needed in v2